src/components/navs/AppNav/AppNavClaimBtn.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <BalPopover no-pad v-if="userClaims != null">
```

Comment:
Currently there is no "loading state" for the claim button, so I'm just hiding it until I can show it. Pinged Pon about this.